src/pe/optional_header.rs

```diff
@@ -120,6 +120,7 @@ pub struct WindowsFields32 {
 }
 
 pub const SIZEOF_WINDOWS_FIELDS_32: usize = 68;
+pub const OFFSET_WINDOWS_FIELDS_32_CHECKSUM: usize = 36;
```

Comment:
Maybe a link to the MSFT docs to derive this?

Reply:
That was me counting bytes from the structure just above. Those offset are not in the original spec.

Reply:
I guess I could use something like https://crates.io/crates/memoffset instead

Reply:
> That was me counting bytes from the structure just above. Those offset are not in the original spec.

I meant the link to the structure

Reply:
does that work for you?

Reply:
Yes, definitely!